scripts/configure-venue-access.ts

```diff
@@ -0,0 +1,141 @@
+import fs from "fs";
+import admin from "firebase-admin";
+import serviceAccount from "./sparkle-7-firebase-adminsdk-f5a3m-482bebb057.json";
+import "firebase/firestore";
+import { Venue } from "../src/types/Venue";
+import {
+  VenueAccessCodes,
+  VenueAccessEmails,
+  VenueAccessType,
+} from "../src/types/VenueAcccess";
+
+const mergeStringArrays = (
+  arr1: string[] = [],
+  arr2: string[] = []
+): string[] => {
+  const result = (arr1 ?? []).map((val) => val.trim());
+  (arr2 ?? []).forEach((val) => {
+    const valTrimmed = val.trim();
+    if (!result.includes(valTrimmed)) {
+      result.push(valTrimmed);
+    }
+  });
+  return result;
+};
+
+function usage() {
+  console.log(`
+${process.argv[1]}: Configure venue access. Supports configuring a secret password, list of emails which can access the venue, or ticket codes.
+
+Usage: npx ts-node ${process.argv[1]} PROJECT_ID VENUE_ID [password|emaillist|codelist] [password | emails file path | codes file path]
+
+Example: npx ts-node ${process.argv[1]} co-reality-map password abc123
+Example: npx ts-node ${process.argv[1]} co-reality-map emails emails-one-per-line.txt
+Example: npx ts-node ${process.argv[1]} co-reality-map codes ticket-codes-one-per-line.txt
+`);
+  process.exit(1);
+}
+
+const argv = process.argv.slice(2);
+if (argv.length < 4) {
+  usage();
+}
+
+const [projectId, venueId, method, accessDetail] = argv;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+if (!(Object as any).values(VenueAccessType).includes(method)) {
+  console.error(`Invalid access method ${method}`);
+  process.exit(1);
+}
```

Comment:
Why do we need to cast `Object` to `any` here?

Reply:
I believe this is a hacky way to create an array from the values of VenueAccessType.

Reply:
```
export enum VenueAccessMode {
  Emails = "emails",
  Codes = "codes",
  Password = "password",
}
```

The way this code is accessing it with `Object.values` seems that `VenueAccessMode` will end up as an object in the compiled code. So we should just be able to use normal ES6 object accessors to check this.

```
if (!VenueAccessType[method]) {
```

Reply:
See https://github.com/sparkletown/sparkle/pull/1016/files?file-filters%5B%5D=.ts#r562441743